tests/integration/integration.go

```diff
@@ -74,7 +74,7 @@ func RunIntegrationTests(t *testing.T) {
 			options := tcase.Setup(t)
 
 			t.Logf("setting up test case")
-			ctx, cancel := context.WithTimeout(context.Background(), 45*time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
Please revert this change. No integration test should be getting close to 45s, and is already a generous upper bound. This is here so that buggy code/tests don't make runs go on forever.